.cursor/scripts/cloud-agent-install.sh

```
@@ -15,39 +15,198 @@ is_true() {
+  sudo ln -sfn "${bindir}/npx" /usr/local/bin/npx
+}
+
+persist_node_path() {
+  local marker="# mattermost-cloud-agent-node-path"
+  local rc="${HOME}/.bashrc"
+  if [ -f "$rc" ] && grep -Fq "$marker" "$rc"; then
+    return 0
   fi
+  {
+    echo "$marker"
+    echo 'export NVM_DIR="$HOME/.nvm"'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)